src-django/api/views.py

```diff
@@ -123,6 +125,55 @@ class ConceptViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
         return models.Concept.objects
 
+    @list_route(methods=['POST'])
+    def import_csv(self, request):
+        if 'csv' not in request.FILES:
+            return JsonResponse(
+                status=status.HTTP_400_BAD_REQUEST,
+                data={
+                    'success': False,
+                    'errors': ["Missing 'csv' parameter"]
+                }
+            )
+
+        csv = request.FILES['csv']
+
+        file_path = "/tmp/{0}.csv".format(str(uuid.uuid4()))
+
+        with open(file_path, 'wb+') as destination:
+            for chunk in csv.chunks():
+                destination.write(chunk)
+
+        try:
+            copy_mapping = CopyMapping(
+                models.Concept,
+                file_path,
+                dict(
+                    created="created",
+                    last_modified="modified",
+                    uuid="uuid",
+                    name="name",
+                    display_name="display_name",
+                    description="description",
+                    data_type="datatype",
+                    mime_type="mimetype",
+                    constraint="constraint"
+                )
+            )
+            copy_mapping.save()
+        except Exception as e:
+            return JsonResponse(
```

Comment:
Addressed in https://github.com/SanaMobile/sana.protocol_builder/commit/b3dfb848362156906ae2a31e508806ef88342729